tests/zfs-tests/tests/functional/cli_root/zpool_split/zpool_split_resilver1.ksh

```diff
@@ -0,0 +1,76 @@
+#!/bin/ksh -p
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2018, Nexenta Systems, Inc.  All rights reserved.
+#
+
+. $STF_SUITE/include/libtest.shlib
+
+#
+# DESCRIPTION:
+# 'zpool split' should fail if resilver in progress for a disk that will not be detached
+#
+# STRATEGY:
+# 1. Create a mirror pool
+# 2. Offline the first VDEV
+# 3. Put some data
+# 4. Online the first VDEV
+# 5. Verify 'zpool split' must fail
+#
+
+verify_runnable "both"
+
+function cleanup
+{
+	destroy_pool $TESTPOOL
+	destroy_pool $TESTPOOL2
+	rm -f $DEVICE1 $DEVICE2
+}
+
+function setup_mirror
+{
+	truncate -s $DEVSIZE $DEVICE1
+	truncate -s $DEVSIZE $DEVICE2
+	log_must zpool create -f $TESTPOOL mirror $DEVICE1 $DEVICE2
+}
+
+log_pass "'zpool split' must fail if resilver in progress for a disk that will not be detached"
+log_onexit cleanup
+
+DEVSIZE='3g'
+DEVICE1="$TEST_BASE_DIR/device-1"
+DEVICE2="$TEST_BASE_DIR/device-2"
+
+ZFS_SCAN_VDEV_LIMIT_SLOW=$((128*1024))
+ZFS_SCAN_VDEV_LIMIT_DEFAULT=$((4*1024*1024))
```

Comment:
You should be able to get the default value with `get_tunable()` instead of hardcoding here the (current) default: i think this is safer in the (unlikely) event this gets changed.

Reply:
I copy-pasted this from a zfs-test, but your suggestion makes sense.